### PR TITLE
Add compatibility document for special relative URI notation NRE.

### PR DESCRIPTION
**Pull request by rmkerr**
This additional page documents a change to `System.Uri` that fixes special relative URI notation when Unicode is present. This change is very unlikely to break things for users, but we're taking a cautious approach when documenting URI changes.

cc: @karelz @davidsh @rpetrusha

### CODE REVIEW

File: Documentation/compatibility/uri-unicode-scheme-only-relative.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Version Introduced
+4.7.3
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+
+<xref:System.Uri> will no longer throw a NullReferenceException when calling <xref:System.Uri.TryCreate> on unusual relative URIs containing Unicode.
```
Comment (davidsh) [inline Documentation/compatibility/uri-unicode-scheme-only-relative.md]: I would change "unusual" to "certain"

Comment (rmkerr) [reply Documentation/compatibility/uri-unicode-scheme-only-relative.md]: Fixed in the latest commit.

File: Documentation/compatibility/uri-unicode-scheme-only-relative.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+## Support special relative URI notation when Unicode is present
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.3
```
Comment (rpetrusha) [inline Documentation/compatibility/uri-unicode-scheme-only-relative.md]: Is this a 4.7.2 or a 4.7.3 change?

File: Documentation/compatibility/uri-unicode-scheme-only-relative.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- The URI must contain percent-encoded Unicode or unreserved symbols.
+
+- [ ] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+Users depending on this behavior to disallow relative URIs should instead specify <xref:System.UriKind.Absolute> when creating a URI.
+
+### Affected APIs
+* `T:System.Uri.TryCreate`
```
Comment (rpetrusha) [inline Documentation/compatibility/uri-unicode-scheme-only-relative.md]: This is an overloaded method, so I think it needs to be \`Overload:System.Uri.TryCreate\`

File: Documentation/compatibility/uri-unicode-scheme-only-relative.md (hunk 2 above)
Comment (davidsh) [reply Documentation/compatibility/uri-unicode-scheme-only-relative.md]: This is supposed to be 4.7.2 I think.

Comment (rmkerr) [reply Documentation/compatibility/uri-unicode-scheme-only-relative.md]: That's correct. I'm getting ahead of myself here on version.

File: Documentation/compatibility/uri-unicode-scheme-only-relative.md (hunk 3 above)
Comment (rmkerr) [reply Documentation/compatibility/uri-unicode-scheme-only-relative.md]: All of these should be addressed in the latest commit. Thanks for the suggestions!

**Pull request merged**